### ISSUE TITLE
Kobotoolbox

### ISSUE THREAD

**Issue by Rabia2219** [civictechstructure/admin#15]

**Comment by Rabia2219** [comment]
#### Messaging for sending out survey
```
The MERL Center is conducting a brief survey on how to collect county data in a Kobotoolbox form. The following are the 3 options that we have designed. Please click on each of the links below and use them to enter dummy data and then indicate your preference in the poll.

```

**Comment by ExperimentsInHonesty** [comment]
```
One of our MERL Center members (Mutsa) has created a Kobotoolbox Survey for Issue #83.  Another member (Luigi), suggested while doing a peer review that instead of an input box, for country, the form could have a drop-down.  Mutsa asked for help creating that, and what we discovered while trying to implement it, is that there are several ways to do it.  We would like to know which way MERL Center members prefer.

tldr; The MERL Center is conducting a brief survey on how to collect county data in a Kobotoolbox form. The following are the 3 options that we have designed. Please click on each of the links below and use them to enter dummy data and then indicate your preference in the poll below the links
1. https://ee.kobotoolbox.org/x/qdMamkIU
2. https://ee.kobotoolbox.org/x/HJQAqiV4
3. https://ee.kobotoolbox.org/x/Aog4FhJu
```

https://github.com/MERLCenter/merl-center/issues/83

**Comment by ExperimentsInHonesty** [comment]
1
2
3
Other (if other, please describe or provide a survey example)

**Comment by ExperimentsInHonesty** [comment]
LinkedIn post
```
The MERL Center is conducting a brief survey on how to collect county/ territory data in a Kobotoolbox forms. The following are the 3 options that we have designed. Please click on each of the links below and use them to enter dummy data and then indicate your preference in the poll 
1. https://ee.kobotoolbox.org/x/qdMamkIU
2. https://ee.kobotoolbox.org/x/HJQAqiV4
3. https://ee.kobotoolbox.org/x/Aog4FhJu
```
